### PR TITLE
finished connect of profile settings and contact info. post part. coz…

### PR DESCRIPTION
**Pull request by DavidSurkov**
now able to send profile page and contacts info to the back end

### CODE REVIEW

File: src/components/signIn/Signin.tsx [hunk 1]
```diff
@@ -18,6 +18,8 @@ import {
 } from './Signin.styles';
 import { useTranslation } from 'react-i18next';
 import { useSignInMutation } from 'service/signinHttp';
+import { useAppDispatch } from '../../redux/hooks';
+import { saveEmail } from '../../redux/reducers/userSlice';
```
Comment (lyudad) [inline src/components/signIn/Signin.tsx]: absolute path. please check you pr before send to me

File: src/components/signUp/signup.tsx [hunk 2]
```diff
@@ -44,8 +47,10 @@ const signUp = () => {
 		} else {
 			await signUp({ email, password })
 				.unwrap()
-				.then(() => {
+				.then(res => {
 					setSucess(true);
+					dispatch(saveUserId(res.id));
```
Comment (lyudad) [inline src/components/signUp/signup.tsx]: save user by id? if yes then its a bad option, we have a token to understand what user is it, so we don't need to path parameter

File: src/pages/settings-page/components/ProfileEdit/hooks/use-on-data-change.ts [hunk 3]
```diff
@@ -132,11 +135,13 @@ export const useOnDataChange = () => {
 		});
 	};
 
-	const [englishOption, setEnglishOption] = useState<string>(PRE_INTERMEDIATE);
+	const [englishOption, setEnglishOption] = useState<EnglishLevelEnum>(
+		EnglishLevelEnum.PRE_INTERMEDIATE,
+	);
 	const englishOptions = [
```
Comment (lyudad) [inline src/pages/settings-page/components/ProfileEdit/hooks/use-on-data-change.ts]: move to utilities, and all text should be in language file

File: src/components/signUp/signup.tsx (hunk 2 above)
Comment (DavidSurkov) [reply src/components/signUp/signup.tsx]: i need this id to send post request

File: src/components/signIn/Signin.tsx (hunk 1 above)
Comment (DavidSurkov) [reply src/components/signIn/Signin.tsx]: done

**Pull request merged**